doc/src/doc-updates/sui-breaking-changes.md

```diff
@@ -32,4 +36,8 @@ This release changes the field value type in `TransactionData` from `gas_payment
 
 **[API breaking change]** - To reduce the size of Sui Full node synchronization payloads, this release removes events from `TransactionEffect`. The events are still included in the `SuiTransactionResponse` returned by `sui_getTransaction` and `sui_submitTransaction` endpoints. For more information, see [PR 7822](https://github.com/MystenLabs/sui/pull/7822).
 
-**[API breaking change]** - The `StakedSui` object now includes the ID of the staking pool, `pool_id`. For more information, see [PR 8371](https://github.com/MystenLabs/sui/pull/8371).
\ No newline at end of file
+**[API breaking change]** - The `StakedSui` object now includes the ID of the staking pool, `pool_id`. For more information, see [PR 8371](https://github.com/MystenLabs/sui/pull/8371).
+
+**[Major breaking change]** The `sui_getObject` endpoint now takes an additional configuration parameter of type `SuiObjectDataOptions` to control which fields the endpoint retrieves. By default, the endpoint retrieves only object references unless the client request  explicitly specifies other data, such as `type`, `owner`, or `bcs`. To learn more, see (https://github.com/MystenLabs/sui/pull/8817)
+
+**[Major breaking change]** - We rewrote ****the ID leak verifier that governs usage of `UID`s in Sui Move code to to flip the logic. With this change, new objects require “fresh” `UID`s created in the function that creates the object. However, when the struct of an object is destroyed, the `UID` can be stored as if the object was wrapped (but without its contents). In contrast, the previous rules stated that the `UID` could come from anywhere, but required destruction when the object was unpacked. We made this change to make using dynamic fields a bit more efficient - you do not always need a `Bag` or `Table` if to retain access to dynamic fields after unpacking an object into its constituent fields. See https://github.com/MystenLabs/sui/pull/8026 for details and a migration example.
```

Comment:
"...We rewrote ****the ID..." Intentional?

Reply:
nope, Notion issue